sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -124,7 +123,7 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
                 Prop2 = 987,
                 Component1 = new Component1
                 {
-                    Metadata = new Component1Metadata { ModelId = componentModelId },
+                    Metadata = new Component1Metadata { },
```

Comment:
not necessary

Reply:
ADT service is expecting Metadata property in the payload, even if it's empty. I tried removing it from the payload and the services rejects it.